### PR TITLE
Add more system sequences

### PR DESCRIPTION
**Pull request by C-D-Lewis**
Also button clicks and random background colors.

Ping @Neal for review.

### CODE REVIEW

**Comment by C-D-Lewis** [comment]
@Neal And this one please!

File: src/main.c [hunk 1]
```diff
@@ -3,18 +3,61 @@
 // Milliseconds between frames
 #define DELTA 13
```
Comment (neal) [inline src/main.c:4]: Where does this come from?

**Comment by neal** [comment]
Did we decide not to publish the other two sequences?

If so, looks good other than the previous comment about the `DELTA` constant.

**Comment by C-D-Lewis** [comment]
Yes we did, and the 13ms appears to be the value I could get closest to the firmware.

**Pull request merged**